adsdata/nonbib.py

```diff
@@ -170,6 +175,14 @@ def get_changed_rows_table(self, table_name, schema_name, meta=None):
                      Column('id', Integer),
                      Column('authors', ARRAY(String)),
                      Column('refereed', Boolean),
+                     Column('ads_openaccess', Boolean),
```

Comment:
I think you might agree that with every new feature this query is getting too long; and we should really use ORM while we can. But there is an issue for that already, so I'm just adding a comment here

(https://github.com/adsabs/AdsDataSqlSync/issues/59)

Reply:
agreed!